cross-chain/arbitrum/hardhat.config.ts

```diff
@@ -0,0 +1,120 @@
+import type { HardhatUserConfig } from "hardhat/config"
+
+import "@keep-network/hardhat-helpers"
+import "@keep-network/hardhat-local-networks-config"
+import "@nomiclabs/hardhat-waffle"
+import "@nomiclabs/hardhat-etherscan"
+import "hardhat-gas-reporter"
+import "hardhat-contract-sizer"
+import "hardhat-deploy"
+import "@typechain/hardhat"
+import "hardhat-dependency-compiler"
+
+const config: HardhatUserConfig = {
+  solidity: {
+    compilers: [
+      {
+        version: "0.8.17",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 1000,
+          },
+        },
+      },
+    ],
+  },
+
+  paths: {
+    artifacts: "./build",
+  },
+
+  networks: {
+    hardhat: {
+      deploy: ["deploy_l2"],
+    },
+    goerliEthereum: {
+      url: process.env.CHAIN_L1_API_URL || "",
+      chainId: 5,
+      deploy: ["deploy_l1"],
+      accounts: process.env.ACCOUNTS_L1_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_L1_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["etherscan"],
+    },
+    mainnetEthereum: {
+      url: process.env.CHAIN_L1_API_URL || "",
+      chainId: 1,
+      deploy: ["deploy_l1"],
+      accounts: process.env.ACCOUNTS_L1_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_L1_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["etherscan"],
+    },
+    goerliArbitrum: {
+      url: process.env.CHAIN_L2_API_URL || "",
+      chainId: 421613,
+      deploy: ["deploy_l2"],
+      accounts: process.env.ACCOUNTS_L2_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_L2_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["arbiscan"],
+      companionNetworks: {
+        l1: "goerliEthereum",
+      },
+    },
+    mainnetArbitrum: {
+      url: process.env.CHAIN_L2_API_URL || "",
+      chainId: 42161,
+      deploy: ["deploy_l2"],
+      accounts: process.env.ACCOUNTS_L2_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_L2_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["arbiscan"],
+      companionNetworks: {
+        l1: "mainnetEthereum",
+      },
+    },
+  },
+
+  external: {
+    deployments: {
+      goerliEthereum: ["./external/goerliEthereum"],
+      goerliArbitrum: ["./external/goerliArbitrum"],
+      mainnetEthereum: ["./external/mainnetEthereum"],
+      mainnetArbitrum: ["./external/mainnetArbitrum"],
+    },
+  },
+
+  etherscan: {
+    apiKey: {
+      goerli: process.env.ETHERSCAN_API_KEY,
+      arbitrumGoerli: process.env.ARBISCAN_API_KEY,
+    },
+  },
+
+  namedAccounts: {
+    deployer: {
+      default: 1,
+      goerliEthereum: process.env.CONTRACT_L1_OWNER_ADDRESS || "",
+      goerliArbitrum: process.env.CONTRACT_L2_OWNER_ADDRESS || "",
+      mainnetEthereum: process.env.CONTRACT_L1_OWNER_ADDRESS || "",
+      mainnetArbitrum: process.env.CONTRACT_L2_OWNER_ADDRESS || "",
```

Comment:
Why don't we just point to `0` account like like we do for `solidity/` deployment scripts [here](https://github.com/keep-network/tbtc-v2/blob/9ef09f055eeb0be8ee9672a908ed38f118d869ad/solidity/hardhat.config.ts#L181-L182)?

Reply:
I'm not sure what was the background behind pointing to `0` in `solidity/`, but I used env. variables to deploy contracts to Goerli from my local box. All I needed was to provide the addresses and priv. keys. I also had to send ETH to these accounts on Goerli Arbitrum to test the deployment.